Sources/Plasma/FeatureLib/pfMetalPipeline/plMetalDevice.h

```diff
@@ -259,20 +259,31 @@ class plMetalDevice
     
     void LoadLibrary();
 
+    void BeginNewRenderPass();
+    void ReleaseSamplerStates();
+    void ReleaseFramebufferObjects();
+
+    // Blur states
+    std::unordered_map<float, NS::Object*> fBlurShaders;
+    
+    // MARK: - Post processing
+private:
     bool NeedsPostprocessing() const
     {
         return fGammaLUTTexture != nullptr;
     }
+    void                      PreparePostProcessing();
+    void                      FinalizePostProcessing();
     void                      PostprocessIntoDrawable();
     void                      CreateGammaAdjustState();
     MTL::RenderPipelineState* fGammaAdjustState;
 
-    void BeginNewRenderPass();
-    void ReleaseSamplerStates();
-    void ReleaseFramebufferObjects();
-
-    // Blur states
-    std::unordered_map<float, NS::Object*> fBlurShaders;
+    // MARK:  - Device capabilities
+public:
+    /// Returns true if the device supports title memory features such as directly writable render buffers.
+    BOOL SupportsTileMemory() { return fSupportsTileMemory; }
```

Comment:
`const`

Also, is this ever used outside of this class? Does it need a public getter vs just accessing `fSupportsTileMemory` directly?

Reply:
Probably does not need to be public. Knowing if the GPU supports tile memory could influence which shader the pipeline picks to do something - but so far most of that has been handled by the device. Especially because the device class has ended up doing a lot of the heavy lifting to make Metal look more like a traditional device to the pipeline.

Reply:
I kept the getter just for cleanliness - but did mark it as inline. I also made it private.

My hunch is someday the pipeline might need it, but it should be fine making it private for now.